src/ReplenishFunction.ts

```diff
@@ -70,7 +97,13 @@ async function defaultReplenishFunction(
         managerBalance.sub(relayServer.config.blockchain.managerMinBalance)
       )
     ) {
-      log.info('Replenishing worker balance by manager rbtc balance');
+      notify(
```

Comment:
@david-iov This will trigger notifications in the format `Notifier | Replenishing worker balance by manager rbtc balance etc...`